docs/sources/enterprise/settings-updates.md

```diff
@@ -36,9 +36,12 @@ For example, if you provide the following `updates`:
 
 ```json
 {
-  "auth.saml": {
-    "enabled": "true",
-    "single_logout": "false"
+  "updates": {
+    "auth.saml": {
+      "enabled": "true",
+      "single_logout": "false"
+    }
+  }
```

Comment:
The payload example was broken (missing closing bracket). I have also added the enclosing `update` property, since without it the payload was confusing, even though we have full payload example below.